manual/installation.tex

```diff
@@ -392,10 +422,11 @@ \subsection{Using vendor optimized numerical libraries, e.g. Intel MKL}
 \subsubsection{Using Intel MKL with non-Intel compilers}
 
 To use Intel MKL with, e.g. an MPICH wrapped gcc:
+\verbatimfont{\footnotesize}
 \begin{verbatim}
 cmake \
  -DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpicxx \
- -DBLA_VENDOR=Intel10_64lp_seq -DCMAKE_PREFIX_PATH=$MKLROOT/lib \
+ -DBLA_VENDOR=Intel10_64lp_seq -DENABLE_MKL -DMKL_ROOT=$MKLROOT/lib \
```

Comment:
I don't think cmake accepts -DENABLE_MKL without a value.
Is just -D DENABLE_MKL=1 sufficient without DBLA_VENDOR and MKL_ROOT?

Reply:
If a MKL_HOME, MKLROOT, or MKL_ROOT environment variable is set, then yes it is.